tests/Common/ComponentsEventsTest.php

```diff
@@ -66,6 +66,11 @@ public function testConfigurationChange()
         $config = $this->getConfiguration();
         $this->components->addConfiguration($config);
 
+        // test no change
+        $this->components->updateConfiguration($config);
```

Comment:
Takhle posíláš jen stejnou konfiguraci
Pro jistotu bych to samý udělal ještě jednou, akorát že pošleš změněný state.

Reply:
přidal sem tam `setState` nebo máš namysli vlastní test case?